scripts/run_commands

```diff
@@ -2,45 +2,84 @@
    export supkg="su-exec"
    export condapkg="Miniconda3-4.5.4-Linux-ppc64le.sh"
    export conda_chksum="05c1e073f262105179cf57920dfc4d43"
+elif [ "$(uname -m)" = "aarch64" ]; then
+   export miniconda_downloaded="1"
+   export supkg="gosu"
+   export python_conda_install=""
+   # defaults is not enough
+   export additional_channel="--add channels c4aarch64"
+   # TODO: Once conda-forge packages these we can remove these flage
+   export conda_install_supkg="0"
+   export conda_install_tini="0"
+else
+   exit 1
 fi
 
 # give sudo permission for conda user to run yum (user creation is postponed
 # to the entrypoint, so we can create a user with the same id as the host)
 echo 'conda ALL=NOPASSWD: /usr/bin/yum' >> /etc/sudoers
 
 # Install the latest Miniconda with Python 3 and update everything.
-curl -s -L https://repo.continuum.io/miniconda/$condapkg > miniconda.sh
-openssl md5 miniconda.sh | grep $conda_chksum
+if [ "${miniconda_downloaded}" != "1" ]; then
+   curl -s -L https://repo.continuum.io/miniconda/$condapkg > miniconda.sh
+   openssl md5 miniconda.sh | grep $conda_chksum
+fi
 bash miniconda.sh -b -p /opt/conda
 rm -f miniconda.sh
 touch /opt/conda/conda-meta/pinned
 ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh
 source /opt/conda/etc/profile.d/conda.sh
 conda activate
 conda config --set show_channel_urls True
-conda config --add channels conda-forge
+conda config ${additional_channel} --add channels conda-forge
+conda config --show-sources
 conda update --all --yes
 conda clean -tipy
 
 # Install conda build and deployment tools.
-conda install --yes --quiet conda-build anaconda-client jinja2 setuptools git "python=3.6"
+conda install --yes --quiet conda-build anaconda-client jinja2 setuptools git ${python_conda_install}
 conda clean -tipsy
 
 # Install docker tools
-conda install --yes $supkg
-export CONDA_SUEXEC_INFO=( `conda list $supkg | grep $supkg` )
-echo "$supkg ${CONDA_SUEXEC_INFO[1]}" >> /opt/conda/conda-meta/pinned
-conda install --yes tini
-export CONDA_TINI_INFO=( `conda list tini | grep tini` )
-echo "tini ${CONDA_TINI_INFO[1]}" >> /opt/conda/conda-meta/pinned
+if [ "${conda_install_supkg}" == "1" ]; then
+   conda install --yes $supkg
+   export CONDA_SUEXEC_INFO=( `conda list $supkg | grep $supkg` )
+   echo "$supkg ${CONDA_SUEXEC_INFO[1]}" >> /opt/conda/conda-meta/pinned
+else
+   if [ "$(uname -m)" = "aarch64" ]; then
+      curl -s -L "https://github.com/tianon/gosu/releases/download/1.11/gosu-arm64" > /opt/conda/bin/gosu
+      chmod +x /opt/conda/bin/gosu
+   else
+      exit 1
+   fi
+fi
+
+if [ "${conda_install_tini}" == "1" ]; then
+   conda install --yes tini
+   export CONDA_TINI_INFO=( `conda list tini | grep tini` )
+   echo "tini ${CONDA_TINI_INFO[1]}" >> /opt/conda/conda-meta/pinned
+else
+   if [ "$(uname -m)" = "aarch64" ]; then
+      curl -s -L "https://github.com/krallin/tini/releases/download/v0.18.0/tini-static-arm64" > /opt/conda/bin/tini
```

Comment:
Can you build tini and su-exec as conda packages and upload them?

Reply:
We have feedstocks for both tini and su-exec. I think there might have been a bit of a bootstrapping problem here originally

Reply:
Yes. We can build them outside of conda-forge infrastructure and upload them. (This is already done for the other packages needed for bootstrap using the c4aarch64 channel.)

Reply:
I can build both and add them to the `c4aarch64` channel.

Reply:
Thanks @jjhelmus. Let us know when they are available. Recipes should be available in conda-forge and all the requirements are there in c4aarch64 already.

Reply:
su-exec and tini packages are now available on the c4aarch64 channel for the linux-aarch64 platform.